Sources/SwiftSyntax/SyntaxText.swift

```diff
@@ -88,8 +91,7 @@ public struct SyntaxText {
     guard !self.isEmpty && !other.isEmpty else {
       return self.isEmpty && other.isEmpty
     }
-    return (other.baseAddress! <= self.baseAddress! &&
-            self.baseAddress! + count <= other.baseAddress! + other.count)
+    return (other.baseAddress! <= self.baseAddress! && self.baseAddress! + count <= other.baseAddress! + other.count)
```

Comment:
@allevato is this one intended because of the line length, or a bug?

Reply:
This is because the current implementation only allows breaking on the left-hand side of an infix operator (with the exception of assignment operators, which may only break on the right). So we remove the break after the `&&` and then continue laying out the rest of the line, which never breaks again because of the high column limit.

With a lower line length, this would "auto-correct" because we would see that the `&&` and the text after it doesn't fit and we would force the break before the `&&` instead, but the high column limit is interfering here.

Reply:
Ah so:
```
    return (other.baseAddress! <= self.baseAddress!
            && self.baseAddress! + count <= other.baseAddress! + other.count)
```
would have remained split?

Reply:
Correct.

Reply:
Is there a setting for that? I personally like them the other way 😅

Reply:
There isn't a configuration option for that today. It probably wouldn't be difficult to add, but to date we haven't had the resources to maintain a large matrix of configurability options and make sure they all work well together, so we mainly prioritize the default style that we use within Google. (PRs welcome 😄)

Reply:
Makes sense, thanks 🙇